src/components/Dapp.re

```diff
@@ -458,7 +458,9 @@ module DefaultLook = {
                  <a
                    href="https://blog.wildcards.world/how-to-buy-a-wildcard-web3-ethereum/"
                    target="_blank">
+                   <span className=Styles.colorGreen>
```

Comment:
We should redesign the "login" screen now and remove this. This text would make more sense integrated in 'login' component. @DenhamPreen , what do you think?

Reply:
Definitely agree. Was on our backlog to put this in a more intuitive place. Also the article deals with the various ways to connect to wallets more that the actual buying process. We should release another video/article showing how to buy an animal (walk through).